README.md

```diff
@@ -13,16 +13,100 @@
 
 <br>
 
-## Usage
+## About
 
-This parser is somewhat generic and robust, it could be used to power any use-case which requires taking TypeScript source code and producing an ESTree-compatiable AST.
+This parser is somewhat generic and robust, and could be used to power any use-case which requires taking TypeScript source code and producing an ESTree-compatiable AST.
 
 In fact, it is already used within these hyper-popular open-source projects to power their TypeScript support:
 
 - [ESLint](https://eslint.org), the pluggable linting utility for JavaScript and JSX
   - See [typescript-eslint-parser](https://github.com/eslint/typescript-eslint-parser) for more details
 - [Prettier](https://prettier.io), an opinionated code formatter
 
+## Installation
+
+```
+npm install --save typescript-estree
+```
+
+## API
+
+### parse(code, options)
+
+Parses the given string of code with the options provided and returns an ESTree-compatible AST. The options object has the following properties:
+
+```javascript
+{
+    // attach range information to each node
+    range: false,
+
+    // attach line/column location information to each node
+    loc: false,
+
+    // create a top-level tokens array containing all tokens
+    tokens: false,
+
+    // create a top-level comments array containing all comments
+    comment: false,
+
+    // enable parsing JSX. For more details, see https://www.typescriptlang.org/docs/handbook/jsx.html
+    jsx: false,
+
+    tolerant: false,
+
+    source: null,
```

Comment:
I wasn't quite sure what this is used for - would love some guidance!

Reply:
It is unused AFAICT